File: aunidos-redirect-back/src/main/java/me/cerratolabs/aunidosredirect/response/ProvinciaResponse.java
package me.cerratolabs.aunidosredirect.response;

import lombok.Data;
import lombok.ToString;
import me.cerratolabs.aunidosredirect.dto.SocialMedia;
import org.springframework.http.HttpStatus;

@Data
@ToString
public class ProvinciaResponse {
    private SocialMedia socialMedia;
    private HttpStatus status;
    private String idTransaccion;
}

File: aunidos-redirect-front/src/app/commons/services/provincia-service/ProvinciaResponse.ts
import {
    ReasonPhrases,
    StatusCodes,
    getReasonPhrase,
    getStatusCode,
} from 'http-status-codes';
import { HttpStatusEnum } from '../../dto/HttpStatus';

export class ProvinciaResponse {
    url: string;
    status: HttpStatusEnum;
    idTransaccion: String;

}